sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs

```diff
@@ -70,15 +77,24 @@ internal ContentType ToContentType()
         internal static SchemaFormat FromContentType(string contentTypeValue)
         {
             var contentEquals = contentTypeValue.Split('=');
-            switch (contentEquals[1])
+            var contentParen = contentTypeValue.Split('/');
```

Comment:
The name for this seems to be misleading, I guess following the pattern would give us `contentSlash` but maybe we could consider renaming to something a bit clearer what they hold?

Reply:
Lol I don't know how I ended up with paren instead of slash, was having a moment I guess.